source/robot_model/src/Model.cpp

```diff
@@ -102,67 +124,80 @@ bool Model::init_qp_solver() {
   // set the initial data of the QP solver_
   this->solver_.data()->setNumberOfVariables(nb_joints + 1);
   this->solver_.data()->setNumberOfConstraints(this->lower_bound_constraints_.size());
-  if (!this->solver_.data()->setHessianMatrix(this->hessian_)) return false;
-  if (!this->solver_.data()->setGradient(this->gradient_)) return false;
-  if (!this->solver_.data()->setLinearConstraintsMatrix(this->constraint_matrix_)) return false;
-  if (!this->solver_.data()->setLowerBound(this->lower_bound_constraints_)) return false;
-  if (!this->solver_.data()->setUpperBound(this->upper_bound_constraints_)) return false;
+  if (!this->solver_.data()->setHessianMatrix(this->hessian_)) { return false; }
```

Comment:
👍

Reply:
is there a reason for this? one liner if statements without brackets are perfectly valid

Reply:
no brackets are still valid, but our style guide recommends against them. Good bracketing is like using parentheses to group order of operations - not strictly necessary but helps readability.

Reply:
I see all good then